plugins/modules/source_control/gitlab/gitlab_user.py

```diff
@@ -107,6 +116,18 @@
       - Define external parameter for this user.
     type: bool
     default: no
+  identities:
+    description:
+      - List of identities to be added/updated for this user.
+    elements: dict
+    type: list
+    version_added: 3.3.0
+  overwrite_identities:
+    description:
+      - Overwrite identities with identities added in this module.
+    type: bool
+    default: no
```

Comment:
```suggestion
    default: false
```

Reply:
All YAML booleans will eventually be changed to `true`/`false`, right now we avoid adding new other instances.